README.md

```diff
@@ -1 +1,46 @@
-todo: write readme
\ No newline at end of file
+# Very Good WearOS App
+
+[![Very Good Ventures][logo_white]][very_good_ventures_link_dark]
+[![Very Good Ventures][logo_black]][very_good_ventures_link_light]
+
+Developed with 💙 by [Very Good Ventures][very_good_ventures_link] 🦄
+
+![coverage][coverage_badge]
+[![style: very good analysis][very_good_analysis_badge]][very_good_analysis_link]
+[![License: MIT][license_badge]][license_link]
+
+A Very Good Flutter Starter Project for WearOS devices created by the [Very Good Ventures Team][very_good_ventures_link].
+
+It includes the basic initial setup and some WearOS specific features to demonstrate how to take advantage of hardware resources.
+
+Generated by the [Very Good CLI][very_good_cli_link] 🤖
```

Comment:
```suggestion
A Very Good Flutter starter project for Wear OS devices created by the [Very Good Ventures Team][very_good_ventures_link].

It includes the basic initial setup and some WearOS specific features to demonstrate how to take advantage of hardware resources.

Generated by the [Very Good CLI][very_good_cli_link] 🤖
```

Reply:
It seems WearOS should be Wear OS (https://en.wikipedia.org/wiki/Wear_OS), so fixed it throughout